static/tests/Layout.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Layout from '../components/Layout.jsx';
+import {shallow} from 'enzyme';
+import ReactTestUtils from 'react-dom/test-utils';
+
+describe('<Layout />', () => {
+  it('should render <Layout />', () => {
+
+  const appList = [{
+    "environments": {
+      "production": {
+        "defaultRef": "master"
+      },
+      "staging": {
+        "defaultRef": "HEAD"
+      },
+    },
+    "id": "1",
+    "name": "freight",
+    "repository": "https://github.com/getsentry/freight.git"
+  }]
+
+  const children = {
```

Comment:
Is this a mock `react-router`? Do we need an actual full-blown mock router? Or can we just pass a `<Component/>`?